src/pathfora.js

```diff
@@ -2264,24 +2264,60 @@
      * @throws {Error} error
      * @param {string} accountId  Lytics account ID
      */
-    recommendContent: function (accountId, filter, callback) {
+    recommendContent: function (accountId, params, callback) {
       // Recommendation API:
-      // https://www.getlytics.com/developers/rest-api/beta#content-recommendation
+      // https://www.getlytics.com/developers/rest-api#content-recommendation
 
-      var recommendUrl,
-          seerId = utils.readCookie('seerid');
+      var seerId = utils.readCookie('seerid');
 
       if (!seerId) {
         throw new Error('Cannot find SEERID cookie');
       }
 
-      recommendUrl = [
+      var recommendParts = [
         '{{apiurl}}/api/content/recommend/',
         accountId,
         '/user/_uids/',
-        seerId,
-        filter ? '?ql=' + filter : ''
-      ].join('');
+        seerId
+      ];
+
+      // construct the query params
```

Comment:
it probably make sense to break this out into a utility function so it can be used elsewhere as we expand / it would be easier to write tests around it